cpp/include/mcts/ActionSelector.hpp

```diff
@@ -32,6 +32,7 @@ struct ActionSelector {
   LocalPolicyArray VN;   // virtual node count
   LocalPolicyArray FPU;  // FPU
   LocalPolicyArray PUCT;
+  float chance_dist[Game::Constants::kMaxBranchingFactor] = {};
```

Comment:
I don't like this living here. I think the branch should happen before constructing an `ActionSelector`.